crates/common/logged_command/src/logged_command.rs

```diff
@@ -259,6 +254,20 @@ impl LoggedCommand {
     }
 }
 
+impl From<tokio::process::Command> for LoggedCommand {
+    fn from(command: Command) -> Self {
+        Self { command }
+    }
+}
+
+impl From<std::process::Command> for LoggedCommand {
+    fn from(command: std::process::Command) -> Self {
+        Self {
+            command: tokio::process::Command::from(command),
+        }
+    }
+}
```

Comment:
note: Had to add this impl because `SudoCommandBuilder` creates `StdCommand`s, and I didn't feel like playing with generics.